### DISCUSSION TITLE
Indonesia vaccination type by date

### DISCUSSION THREAD

**Discussion by dant89** [owid/covid-19-data#2829] [Q&A] [answered]
In the Indonesia vaccination dataset: https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/country_data/Indonesia.csv

The `vaccine` column contains combined values where multiple vaccines are added in a linear fashion as they became available.

Is there any way to generate this data so the values are separated? For example to get the report values of two different vaccination administrations on one date?

**Comment by lucasrodes** [comment, marked as answer]
Hi @dant89, and thanks for your interest in the project!

We only report vaccination doses by manufacturer for a subset of countries. We don't plan to extend this to other countries for now.

You can find the file with all the data here: https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/vaccinations-by-manufacturer.csv
